tests/playwright/examples/test_shiny_create.py

```diff
@@ -48,7 +48,10 @@ def test_template_examples(page: Page, ex_app_path: str) -> None:
 
 
 @pytest.mark.flaky(reruns=reruns, reruns_delay=reruns_delay)
-@pytest.mark.parametrize("app_template", ["basic-app", "dashboard", "multi-page"])
+@pytest.mark.parametrize(
+    "app_template",
+    ["basic-app", "basic-sidebar", "dashboard", "dashboard-tips", "basic-navigation"],
+)
```

Comment:
Can get we the folder names from the original folder location? 
And can we also assert their length is > 0?